scripts/README.md

```diff
@@ -2,22 +2,19 @@
 
 This repository contains scripts that are used by multiple release steps.
 
-<b>Note:</b> Make sure you have a `~/.my.cnf` file populated with `mysqldump` and `mysql` parameters to use this script. See https://stackoverflow.com/a/24369804
+<h3>createOrReplaceDatabase.sh</h3>
 
 
-<h3>createOrReplaceDatabase.sh</h3>
+<b>Note:</b> Make sure you have a `.my.cnf` file populated, in your <b>home directory</b>, with `mysqldump` and `mysql` parameters to use this script. See https://stackoverflow.com/a/24369804
```

Comment:
Do you want to mention that the `my.cnf` parameters accepted include "user" and "password" with optional "host" and "port"?